### PR TITLE
cn, event: add node/cn/api_backend_test.go

### PR DESCRIPTION
**Pull request by ehnuje**
## Proposed changes

- To increase the coverage of `cn/api_backend.go`,`node/cn/api_backend_test.go` is added
- `MockSubscription` is added to mock `Subscription` interface for testing

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by ehnuje** [comment]
@jeonghonah @nogang @KimKyungup Dear reviewers, could you please review this change?

File: node/cn/api_backend_test.go [hunk 1]
```diff
@@ -0,0 +1,544 @@
...
+
+	mockTxPool.EXPECT().SubscribeNewTxsEvent(txCh).Return(sub).Times(1)
+
+	api.SubscribeRemovedLogsEvent(rmCh)
+	api.SubscribeChainEvent(ceCh)
+	api.SubscribeChainHeadEvent(chCh)
+	api.SubscribeChainSideEvent(csCh)
+	api.SubscribeLogsEvent(leCh)
+
+	api.SubscribeNewTxsEvent(txCh)
```
Comment (KimKyungup) [inline node/cn/api_backend_test.go]: Why don't you check the returned `sub` is equal with given `sub` in EXPECT routine?

**Comment by ehnuje** [comment]
@KimKyungup I'm planning to, but not in this PR.

**Comment by ehnuje** [comment]
@jeonghonah @nogang Could you please review this change?

**Comment by whoisxx** [comment]
I changed klaytn PR base branch master to `dev` for CICD workflow

**Comment by ehnuje** [comment]
@jeonghonah @nogang Could you please review this change?

**Comment by ehnuje** [comment]
@jeonghonah @nogang Could you please review this change?

**Pull request merged**